django/views/debug.py

```diff
@@ -125,6 +125,8 @@ def cleanse_setting(self, key, value):
 
         if is_sensitive:
             cleansed = self.cleansed_substitute
+        elif key == settings.SESSION_COOKIE_NAME:
+            cleansed = self.cleansed_substitute
```

Comment:
Can we move this branch into the `is_sensitive` check above? 

```
            is_sensitive = (
                self.hidden_settings.search(key) or key == settings.SESSION_COOKIE_NAME
            )
```

Reply:
I changed the code. It was a bit more complicated because of the `try … except`. I hope the result matches what you had in mind.